packages/extension/src/components/pages/Login/Login.tsx

```diff
@@ -61,12 +61,21 @@ import { useWallet } from '../../../contexts';
 import { useKeyUp } from '../../../hooks/useKeyUp';
 import { loadData } from '../../../utils';
 import { loadRememberSessionState, saveRememberSessionState } from '../../../utils/login';
+import {
+  loadFromChromeLocalStorage,
+  saveInChromeLocalStorage
+} from '../../../utils/storageChromeLocal';
 import { Logo } from '../../atoms/Logo';
 
+const maxAttempts = 5;
```

Comment:
It would be nice to have this constant in uppercase in order to follow the convention.
MAX_ATTEMPTS

Reply:
Was not familiar with the convention, sorry